### ISSUE TITLE
[Accessibility]: Cannot navigate to file path links using keyboard

### ISSUE THREAD

**Issue by varungandhi-src** [sourcegraph/sourcegraph#36628]
### Audit type

Keyboard navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/34448

### Problem description

On top of the blob view are path links. It doesn't seem possible to navigate to these links using keyboard navigation. The Copy path to clipboard button can be reached alright.

### Expected behavior

It should be possible to reach the path links.

### Assigning labels

- [ ] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by umpox** [comment]
@varungandhi-src I tried to replicate this at https://sourcegraph.com/github.com/sourcegraph/sourcegraph/-/blob/client/web/src/components/FilteredConnection/ConnectionNodes.tsx?L118

AFAIK navigation works OK, might have been fixed recently. Feel free to reopen if you can replicate!

![PathNav](https://user-images.githubusercontent.com/9516420/176178161-37f3da1a-f42a-46e3-84e1-9b5319e29dee.gif)

**Comment by varungandhi-src** [comment]
Did you try the above in Chrome? It works for me in Chrome (with Tab / Shift+Tab), but it doesn't work in Safari or Firefox. Attached screen recording below for Firefox 101.0.1.

https://user-images.githubusercontent.com/93103176/176386461-75e456f1-7a10-49a9-af07-8e5bb5ab32d1.mov

**Comment by umpox** [comment]
@varungandhi-src I did! thanks will take a look!

**Comment by umpox** [comment]
Have investigated further, I also discovered the same issue on every webpage using Safari and Firefox. It turns out this is intentional, and should be enabled manually https://www.scottohara.me/blog/2014/10/03/link-tabbing-firefox-osx.html

I don't think there's anything we can/should do here to override this
